content/models.py

```diff
@@ -27,3 +27,17 @@ class OurStoryTitle(models.Model):
 
     def __str__(self):
         return self.title
+
+class ChildrensProgram(models.Model):
+	pagetitle = models.CharField(blank=True, null=True, max_length=100)
```

Comment:
either they have title or subtitle. The page title is at the top of the page once, and the subtitle is for each section. So they have to have one or the other. Would it be better : to make a new model, don't let them edit the page title, keep it how it is or ..something else?